samples/bluetooth/hci_lpuart/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _bluetooth-hci-lpuart-sample:
+
+Bluetooth: HCI Low Power UART
+#############################
+
+Sample is based on :ref:`zephyr:bluetooth-hci-uart-sample` but is using
+:ref:`_uart_nrf_sw_lpuart` for HCI UART communication.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf52840
+
+Overview
+********
+
+Sample implements Bluetooth HCI controller using :ref:`_uart_nrf_sw_lpuart` for
+UART communication.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/hci_lpuart`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+Programming the sample
+======================
+
+In order to program the sample on nRF9160DK put the **SW5** switch (marked debug/prog)
+in the **NRF52** position.
+
+Testing
+=======
+
+Does not apply. Testing depends on host application.
+
+Dependencies
+************
+
+This sample uses the following |NCS| driver:
+
+* :ref:`uart_nrf_sw_lpuart`
+
+From Zephyr
+  * :ref:`zephyr:bluetooth-hci-uart-sample`
```

Comment:
I suppose as an improvement we could extract the controller-side driver to an actual driver so that we don't have to re-use the sample source.
I've made a note of it.